src/Pages/CountryDetail.jsx

```diff
@@ -1,151 +1,172 @@
-
-                    <div className="mb-6 flex flex-col gap-2 lg:mb-0 lg:col-start-2 lg:row-span-2">
-                        <h1 className="font-bold text-3xl mb-4">{countriesData.name}</h1>
-
-                        <p>
-                            Native Name:
-                            <span className="opacity-60">{countriesData.nativeName}</span>
-                        </p>
-                        <p>
-                            Population:
-                            <span className="opacity-60"> {countriesData.population}</span>
-                        </p>
-                        <p>
-                            Region:
-                            <span className="opacity-60"> {countriesData.region}</span>
-                        </p>
-                        <p>
-                            Sub Region:
-                            <span className="opacity-60"> {countriesData.subRegion}</span>
-                        </p>
-                        <p>
-                            Capital:
-                            <span className="opacity-60"> {countriesData.capital}</span>
-                        </p>
-                    </div>
-
-                    <div className="mb-6 flex flex-col gap-2 lg:mb-0 lg:col-start-3 lg:row-span-2">
-                        <p>
-                            Top Level Domain:
-                            <span className="opacity-60"> {countriesData.topLevelDomain}</span>
-                        </p>
-                        <p>
-                            Currencies:
-                            <span className="opacity-60"> {countriesData.currency}</span>
-                        </p>
-                        <p>
-                            Languages:
-                            <span className="opacity-60"> {countriesData.languages}</span>
-                        </p>
-                    </div>
-
-
-
-                    <div className="flex flex-wrap gap-3 lg:items-center lg:col-start-2 lg:row-span-2 lg:place-content-center">
-                        <h2 className="text-lg mb-2">Border Countries:</h2>
-
-                        <Link className="bg-light-theme-elements dark:bg-dark-theme-elements shadow-md p-2"
-                            to={`/Details/${countriesData.name}`}>
-
-                        </Link>
-                    </div>
-
-
-                </div>
-            </main >
-
-            <footer className="text-center text-balance p-4">
-                <p className="text-sm">
-                    Challenge by <a className="text-blue-700" href="frontendmentor.io">Frontend Mentor.
-                    </a> coded by <a className="text-blue-700" href="https://github.com/Mazz100">Mazen Hassan.</a>
-                </p>
-            </footer>
-        </div >
-
-    );
+  const countryDetail = useLoaderData();
```

Comment:
I've removed unused variables here and used a clearer name for the value returned from the loader